### PR TITLE
add serviceTypeUuid property for filtering bills

### PR DESCRIPTION
**Pull request by Injiri**
This should enable filtering or billing history by serviceTypes

### CODE REVIEW

**Comment by Injiri** [comment]
@ojwanganto help review

File: api/src/main/java/org/openmrs/module/kenyaemr/cashier/api/model/BillLineItem.java [hunk 1]
```diff
@@ -31,6 +31,7 @@ public class BillLineItem extends BaseOpenmrsData {
 	private StockItem item;
 	private BillableService billableService;
 	private String itemOrServiceConceptUuid;
+	private String serviceTypeUuid;
```
[the remark this answers is not in the document]
Comment (Injiri) [reply api/src/main/java/org/openmrs/module/kenyaemr/cashier/api/model/BillLineItem.java:34]: The itemorservice uuid is more of the concept associated with the billable service or item.
The service Type uuid will give info on the type of service(categorization) I.e all services with concept class test will be under lab service as a type

**Comment by amosmachora** [comment]
@ojwanganto any progress on this. There is a feature I am working on on the frontend that relies on this.

**Comment by ojwanganto** [comment]
@amosmachora @Injiri - we should at least discuss the expectations so that we can agree on the designs. This should in no way change what you have done but to help you work on this much faster.
@Injiri - is the property synonymous to the department? Do we already save it in the database?

**Comment by amosmachora** [comment]
The essence of this feature is we had a checklist of the blockers that would prevent migrating to 3.x in the Oloitoktok facility. One of the blockers was a missing way to view bills by `department`. Since on 3.x we do not have the concept of department, we went with the service type approach since it would serve the same purpose. i.e bills with a service type of `lab service or `pharmacy service` can attributed to that 'department'. 

So what I have done is I have added a new filter by service type on the payment history page to achieve this. I currently have a way of achieving this without this PR but it is not performant. So @Injiri opted to add that field to help me out.

**Comment by ojwanganto** [comment]
That makes sense. I will call @Injiri for alignment. We can discuss also the concept of departments

**Review by ojwanganto — APPROVED** [review]
Thanks, @Injiri

**Pull request merged**